xapian-letor/tests/.gitignore

```diff
@@ -22,3 +22,7 @@
 /perflog.xml
 /submitperftest
 /zlib-vg.so
+/listnet_ranker_training.txt
+/svm_ranker_training.txt
+/api_letor.h
+/scorer_output.txt
```

Comment:
It's perhaps not entirely obvious, but the existing list groups similar files together, and is sorted within each group - so it'd be better to add `/api_letor.h` to the end of the existing group of `/api_*` files above.